packages/jest-config/src/utils.js

```diff
@@ -66,9 +67,11 @@ export const replaceRootDirInPath = (
     return filePath;
   }
 
-  return path.resolve(
-    rootDir,
-    path.normalize('./' + filePath.substr('<rootDir>'.length)),
+  return slash(
```

Comment:
I feel like this could break glob patterns with escape sequences.

Like `<rootDir>/test \(in brakets\)/*.js`

Maybe add a test case for this.